xfce/session.sh

```diff
@@ -19,6 +20,31 @@ destdir="$(xdg_config_home)/xfce4/xfconf/xfce-perchannel-xml"
 mkdir -p "${destdir}"
 cp /etc/xdg/xfce4/panel/default.xml "${destdir}"/xfce4-panel.xml
 
+install_geometry_script() {
+  local f destdir geom_sh
+
+  f="clusterware-geometry.sh"
+  if ! geom_sh=$(xdg_data_search clusterware/bin/$f); then
+      destdir="$(xdg_data_home)/clusterware/bin"
+      geom_sh="${destdir}/${f}"
+      mkdir -p "${destdir}"
+      cp "${cw_ROOT}"/etc/sessions/xfce/${f} "${geom_sh}"
+      chmod 755 "${geom_sh}"
+  fi
+
+  f="clusterware-geometry.desktop"
+  if ! xdg_config_search autostart/$f; then
+      destdir="$(xdg_config_home)/autostart"
+      mkdir -p "$destdir"
+      cp "${cw_ROOT}"/etc/sessions/xfce/${f}.tpl "${destdir}/${f}"
+      sed -i -e "s,_CLUSTERWARE_GEOMETRY_SH_,${geom_sh},g" "${destdir}/${f}"
+  fi
+}
+
+if distro_is el7; then
```

Comment:
I would imagine that XFCE behaviour is the same between EL7 and EL6.  In other places behaviour does vary (GNOME in EL6 is a different version from GNOME in EL7).

It's not a big deal but probably worth dropping this condition IMO.